.gitignore

```diff
@@ -118,3 +118,6 @@ env/
 
 ### Configuration files ###
 .env
+
+### Trello API keys ###
+auth.py
```

Comment:
Word of warning - moving the variables to environment variables (by adding them to the .env file) would be more convenient for later modules and the exercise instructions will assume that's what you're doing.